components/LobbyCircle.tsx

```diff
@@ -1,31 +1,31 @@
 import React from 'react';
-import styles from '../styles/home.module.scss';
+import styles from '../styles/LobbyCircle.module.scss';
+
+enum Create {
+  NotCreate,
+  CreateOnly,
+  CreateWithLobbies
+}
 
 export interface LobbyCircleProps {
   name: string;
-  // create options
-  // 0 = not create
-  // 1 = create with no lobbies
-  // 2 = create with lobbies
   create?: number;
-  offset?: boolean;
 }
 
-export function LobbyCircle({name, create = 0, offset = false}: LobbyCircleProps): JSX.Element {
+export function LobbyCircle({name, create = Create.NotCreate}: LobbyCircleProps): JSX.Element {
+
   let finalStyle;
 
-  if (create == 1) {
-    finalStyle = `${styles.circle} ${styles.single}`;
-  } else if (create == 2) {
-    finalStyle = `${styles.circle} ${styles.create}`;
-  } else if (offset) {
-    finalStyle = `${styles.circle} ${styles.offset}`;
+  if (create == Create.CreateOnly) {
```

Comment:
turn this into a map!

```suggestion
  const CreateOptions2Style = {
    [CreateOptions.CreateOnly]: styles.single
    ....
  }
```

Reply:
good idea!